xbmc/pvr/windows/GUIWindowPVRRecordings.cpp

```diff
@@ -262,11 +262,33 @@ bool CGUIWindowPVRRecordings::OnMessage(CGUIMessage &message)
           {
             case ACTION_SELECT_ITEM:
             case ACTION_MOUSE_LEFT_CLICK:
+              switch(CSettings::GetInstance().GetInt(CSettings::SETTING_MYVIDEOS_SELECTACTION))
+              {
+                case SELECT_ACTION_CHOOSE:
+                  OnPopupMenu(iItem);
+                  bReturn = true;
+                  break;
+                case SELECT_ACTION_PLAY_OR_RESUME:
+                  PlayFile(m_vecItems->Get(iItem).get(), false, true);
```

Comment:
Perhaps the value of `m_vecItems->Get(iItem).get()` could be stored before the `switch` statement begins since it's used 3-4 times?